Sources/Realtime/V2/RealtimeChannelV2.swift

```diff
@@ -12,6 +12,7 @@ import Helpers
 public struct RealtimeChannelConfig: Sendable {
   public var broadcast: BroadcastJoinConfig
   public var presence: PresenceJoinConfig
+  public var privateTopic: Bool
```

Comment:
Although `private` is a swift keyword, you can use it as a property name if you escape it with `, example:

```suggestion
  public var `private`: Bool
```

Or maybe we could name it `isPrivate` to avoid the escaping?

Reply:
I like that! isPrivate is a great name that I should have thought about 😅